model-optimizer/extensions/front/onnx/MVNNormalize.py

```diff
@@ -0,0 +1,48 @@
+"""
+ Copyright (C) 2017-2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+from extensions.ops.mvn import MVN
+from extensions.ops.range import Range
+from extensions.ops.rank import Rank
+from mo.front.common.partial_infer.utils import int64_array
+from mo.front.common.replacement import FrontReplacementPattern
+from mo.front.tf.graph_utils import create_op_with_const_inputs
+from mo.graph.graph import Graph, rename_nodes
+
+import numpy as np
+
+
+class MvnOnnxToMvn(FrontReplacementPattern):
+    """
+    Replace AttributedMVN operation from ONNX with MVN
+    """
+    enabled = True
+
+    def find_and_replace_pattern(self, graph: Graph):
+        for node in graph.get_op_nodes(op='AttributedMVN'):
+            node_name = node.soft_get('name', node.id)
+
+            axes = node.soft_get('axes', np.array([0, 2, 3], dtype=np.int64))
```

Comment:
The `axis` attribute is always set in the `MeanVarianceNormalizationExtractor` so no need to set some default value here.